### PR TITLE
Login Form Container

### PR DESCRIPTION
**Pull request by bransometan**
# Pull Request

## Description
[Provide a brief description of the changes in this pull request.]

## Related Issues
[Reference any related issues that this pull request addresses or resolves.]

## Checklist
Please review and check the following before submitting your pull request:

- [ ] I have read the [CONTRIBUTING.md](../CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ ] I have written tests for my changes (if applicable).
- [ ] All new and existing tests passed.
- [ ] I have updated the documentation accordingly (if applicable).
- [ ] I have added comments to my code, particularly in areas that may be confusing.

## Screenshots (if applicable)
[Include any relevant screenshots or GIFs to visually demonstrate the changes.]

### Desktop
image

### Mobile
image

## Additional Information
[Include any additional information that might be helpful for reviewers or maintainers.]
Not sure if I am doing it correctly. Do update me if changes is required. I think the height of the inner component depends on the size of the 2 components in login form.

## Reviewer Notes
[Optional: Include specific instructions or notes for reviewers, if necessary.]

### CODE REVIEW

**Comment by gnimnix** [comment]
hi @bransometan , please request me as reviewer and merge into `epic/DEV-3-authentication` instead of `main`!

File: README.md
Comment (gnimnix) [inline README.md:1]: this file should not be changed

File: src/assets/css/App.css
Comment (gnimnix) [inline src/assets/css/App.css:1]: this file should not be changed

File: src/assets/css/index.css
Comment (gnimnix) [inline src/assets/css/index.css:1]: this file should not be changed

File: src/assets/css/reset.css
Comment (gnimnix) [inline src/assets/css/reset.css:1]: this file should not be changed

File: vite.config.ts.timestamp-1707232163595-fbad76be9c0df.mjs
Comment (gnimnix) [inline vite.config.ts.timestamp-1707232163595-fbad76be9c0df.mjs:1]: what is this file for?

**Review by gnimnix — CHANGES_REQUESTED** [review]

**Comment by gnimnix** [comment]
@bransometan think you changed some files that should not be changed and committed the changes already. Suggest you create another branch from `epic/DEV-3-authentication`, you can call it `TAS-2-login-form-containers-edited`. Then copy the files over.

**Comment by bransometan** [comment]
Have edited the changes and submitted another PR

**Pull request closed without merging**